cmd/generate-schema.go

```diff
@@ -44,17 +34,26 @@ func GenerateSchema() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			s := schema.NewJsonSchema()
 
-			json, err := s.ToJSON()
+			if outputFilePath == "-" {
```

Comment:
Thanks @maroshii, changes done with https://github.com/okteto/okteto/pull/4621/commits/15aa4b007e446891bc1b2f9af5b7fff3b48f528c ✅ 

nit: I've removed the log in case of log to file, but it's a continuous decision as it's more consistent and a common practice not to log in these cases, I think.